### PR TITLE
Binance account registration link fix

### PR DESCRIPTION
**Pull request by Ludoshales**
Fixed the link to register for Binance, in the README section

### CODE REVIEW

**Comment by philipodev** [comment]
indeed the current link doesnt work. but why add the referal code to it?

**Comment by Ludoshales** [comment]
> indeed the current link doesnt work. but why add the referal code to it?

Oh sorry I didn't see that

**Comment by Ludoshales** [comment]
I don't understand why I got so many thumbs down?

**Comment by philipodev** [comment]
> I don't understand why I got so many thumbs down?

because people (including me) thinks that you're trying to make money of you're pull request, by referal link.

**Comment by SaLuci** [comment]
> I don't understand why I got so many thumbs down?

if someone should make money out of it ,it should be edeng23

**Comment by Ludoshales** [comment]
oh I understand. This is my first contribution to open source ever. I just saw a bug that I thought I could fix. I'm not trying to gain anything from it honestly. If anyone would like to chat with me over discord or something I'm more than happy to clear anything up

**Comment by philipodev** [comment]
> oh I understand. This is my first contribution to open source ever. I just saw a bug that I thought I could fix. I'm not trying to gain anything from it honestly. If anyone would like to chat with me over discord or something I'm more than happy to clear anything up

just remove this `?ref=EHEAMGSD` from the link and it will be fine

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The bot jumps between a configured set of coins on the condition that it does no
```
Comment (nanashj) [inline README.md:29]: []()

**Review by nanashj — APPROVED** [review]

**Comment by nanashj** [comment]
[]()

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The bot jumps between a configured set of coins on the condition that it does no
 
 ## Binance Setup
 
-* Create a [Binance account](https://www.binance.com/hw_register.html).
+* Create a [Binance account](https://accounts.binance.com/en/register?ref=EHEAMGSD).
```
Comment (nanashj) [inline README.md:32]: #

**Pull request closed without merging**